sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -146,6 +164,13 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_sdw_machines[] = {
 		.sof_fw_filename = "sof-tgl.ri",
 		.sof_tplg_filename = "sof-tgl-rt5682.tplg",
 	},
+	{
+		.link_mask = 0x3, /* rt5682 on link0 and 2xmax98373 on link 1 */
+		.links = tgl_chromebook_base,
+		.drv_name = "sof_sdw",
+		.sof_fw_filename = "sof-tgl.ri",
+		.sof_tplg_filename = "sof-tgl-rt5682-mx8373.tplg",
```

Comment:
Can you please append sdw to tplg name for easy identification?

Reply:
Ack, shall make it as sof-tgl-sdw-rt5682-mx8373.tplg